spec/features/step_by_step_navigation_spec.rb

```diff
@@ -26,37 +26,57 @@
       and_I_see_my_selected_preferences
     end
 
-    scenario "User schedules publishing" do
+    scenario "User cannot see Schedule button without Scheduling permissions" do
       given_there_is_a_draft_step_by_step_page
-      and_I_visit_the_scheduling_page
-      and_I_fill_in_the_scheduling_form
-      when_I_submit_the_form
-      then_I_should_see "has been scheduled to publish"
-      and_the_step_by_step_should_have_the_status "Scheduled"
-      and_there_should_be_a_change_note "Minor update scheduled by Test author for publishing at #{schedule_time}"
+      and_I_visit_the_publish_or_delete_page
+      then_there_should_be_no_schedule_button
     end
 
-    scenario "User tries to schedule publishing for date in the past" do
-      given_there_is_a_draft_step_by_step_page
-      and_I_visit_the_scheduling_page
-      and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
-      when_I_submit_the_form
-      then_I_should_see "Scheduled at can't be in the past"
-      and_the_step_by_step_should_have_the_status "Draft"
+    context "Given I have Scheduling permissions" do
+      before do
+        given_I_have_scheduling_permissions
+      end
+
+      scenario "User schedules publishing" do
+        given_there_is_a_draft_step_by_step_page
+        and_I_visit_the_scheduling_page
+        and_I_fill_in_the_scheduling_form
+        when_I_submit_the_form
+        then_I_should_see "has been scheduled to publish"
+        and_the_step_by_step_should_have_the_status "Scheduled"
+        and_there_should_be_a_change_note "Minor update scheduled by Test author for publishing at #{schedule_time}"
+      end
+
+      scenario "User tries to schedule publishing for date in the past" do
+        given_there_is_a_draft_step_by_step_page
+        and_I_visit_the_scheduling_page
+        and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
+        when_I_submit_the_form
+        then_I_should_see "Scheduled at can't be in the past"
+        and_the_step_by_step_should_have_the_status "Draft"
+      end
+
+      scenario "User tries to schedule publishing for an already scheduled step by step" do
+        given_there_is_a_scheduled_step_by_step_page
+        when_I_visit_the_scheduling_page
+        then_I_should_see "Scheduled to be published at"
+        and_there_should_be_no_schedule_button
+      end
     end
+  end
 
-    scenario "User tries to schedule publishing for an already scheduled step by step" do
-      given_there_is_a_scheduled_step_by_step_page
-      when_I_visit_the_scheduling_page
-      then_I_should_see "Scheduled to be published at"
-      and_there_should_be_no_schedule_button
-    end
+  def given_I_have_scheduling_permissions
+    stub_user.permissions << "Scheduling"
   end
 
   def and_I_visit_the_navigation_rules_page
     visit step_by_step_page_navigation_rules_path(@step_by_step_page)
   end
 
+  def and_I_visit_the_publish_or_delete_page
```

Comment:
This step already exists in `managing_step_by_step_pages_spec.rb`. If these tests are moved there, the existing `the_publish_or_delete_page` could be reused.